app/coffeescript/components/ui/base_map.coffee

```diff
@@ -91,16 +78,19 @@ define [
 
     @fireOurMapEvents = () ->
       eventsHash = @attr.gMapEvents
-      if @attr.infoWindowOpen == true
+      if @attr.infoWindowOpen is true
         eventsHash['center_changed'] = false
         eventsHash['max_bounds_changed'] = false
-      clearInterval(@intervalId)
 
       if eventsHash['max_bounds_changed']
         @trigger document, 'uiMapZoomForListings', @mapChangedData()
         @trigger document, 'uiInitMarkerCluster', @mapChangedData()
         @trigger document, 'mapRendered', @mapChangedData()
         @trigger document, 'uiNeighborhoodDataRequest', @mapChangedDataBase()
+      else if eventsHash['zoom_changed']
+        @trigger document, 'uiMapZoom', @mapChangedData()
+      else if eventsHash['center_changed']
+        @trigger document, 'uiMapCenter', @mapChangedData()
```

Comment:
Should these just be plain `if` statements (instead of `else if`s)?
In the case where both have occurred, only the `uiMapZoom` will be emitted so listeners for `uiCenterChanged` will be clueless.

I realize this isn't useful for map_variation_two (in fact, it would cause an extra pin request).

Or perhaps trigger a single event which notes whether zoom and/or center changed. Something like:
```
if eventsHash['zoom_changed'] || eventsHash['center_changed']
  payload = @mapChangedData()
  payload['zoom_changed'] = eventsHash['zoom_changed']
  payload[center_changed'] = eventsHash['center_changed']
  @trigger document, 'uiMapChanged', payload
```

Very open to a better name than "uiMapChanged"; it was just the first thing I thought of.